BusinessCore.Tests/Celia_FunctionalProgramingTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Linq;
+using BusinessCore.Tests.Services;
+using CarManagement.Core.Models;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace BusinessCore.Tests
+{
+    [TestClass]
+    public class Celia_FunctionalProgramingTests
+    {
+        private readonly ArrayVehicleStorage vehicleStorage;
+
+        public Celia_FunctionalProgramingTests()
+        {
+            this.vehicleStorage = new ArrayVehicleStorage();
+        }
+
+        [TestMethod]
+        public void get_the_enrollment_and_horsePower_of_the_vehicles_ordered_descending_by_horsePower()
+        {          
+
```

Comment:
Please, add vehicles variable of type IVehicle[].

Reply:
Compile solution